src/features/controls.ts

```diff
@@ -143,7 +146,10 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
 
     protected[$promptElement] =
         this.shadowRoot!.querySelector('.controls-prompt')!;
+    protected[$promptElementSVG] =
```

Comment:
Changed so that we no longer depend on the SVG element (https://github.com/GoogleWebComponents/model-viewer/pull/788/commits/c7dfc339dcdfe09cfe894cd9b02a784127e1e885), Since the opacity is being animated on the slot itself removing the class would suddenly fade out the prompt element. I added a `div.wrapper` to solve this. If there's a better way to do this let me know! Tested out with an `img` and works correctly, so the icon is customizable via slots.